src/resources/views/crud/fields/summernote.blade.php

```diff
@@ -43,7 +44,7 @@
 {{-- FIELD JS - will be loaded in the after_scripts section --}}
 @push('crud_fields_scripts')
     {{-- include summernote js --}}
-    @loadOnce('packages/summernote/dist/summernote-bs4.min.js')
+    @basset('https://unpkg.com/summernote@0.8.20/dist/summernote.min.js')
     @loadOnce('bpFieldInitSummernoteElement')
```

Comment:
Fields are still using `@loadOnce()`. We should probably make them use `@bassetBlock()`, so that there's absolutely no mention of `@loadOnce()` anywhere. Only THEN can we remove the `digitallyhappy/assets` dependency. 

Devs will still be able to use it, if they want/need it. But not Backpack itself.

Reply:
Fixed in #4933